src/main/scala/com/softwaremill/realworld/profiles/ProfilesEndpoints.scala

```diff
@@ -0,0 +1,41 @@
+package com.softwaremill.realworld.profiles
+
+import com.softwaremill.realworld.common.*
+import sttp.tapir.generic.auto.*
+import sttp.tapir.json.zio.jsonBody
+import sttp.tapir.server.ServerEndpoint.Full
+import sttp.tapir.ztapir.*
+import sttp.tapir.{EndpointInput, PublicEndpoint, Validator}
+import zio.{Cause, Console, Exit, IO, ZIO, ZLayer}
+
+import scala.util.chaining.*
+
+class ProfilesEndpoints(base: BaseEndpoints, profilesService: ProfilesService) {
+
+  private val getProfile: ZServerEndpoint[Any, Any] = base.secureEndpoint.get
+    .in("api" / "profiles" / path[String]("username"))
+    .out(jsonBody[Profile])
+    .serverLogic(session => username => profilesService.getProfile(username, session.email) pipe defaultErrorMappings)
```

Comment:
let's use dot consistently, so `.pipe`; plus, do other endpoints follow the same pattern?

Reply:
It's a minor but I would also add new lines to be consistent with other files (and in my opinion this is a clearer approach)

Reply:
for that we should simply rely on scalafmt :)